addons/captives/stringtable.xml

```diff
@@ -136,6 +136,23 @@
             <Chinesesimp>将俘虏带出载具</Chinesesimp>
             <Turkish>Tutukluyu indir</Turkish>
         </Key>
+        <Key ID="STR_ACE_Captives_BlindfoldCaptive">
+            <English>Blindfold Captive</English>
+            <German>Augen verbinden</German>
+            <Spanish>Vendas los ojos al cautivo</Spanish>
+            <French>Bandes les yeux</French>
+            <Polish>zawiązujesz oczy jeńcowi</Polish>
+            <Czech>zavážete zajatci oči</Czech>
+            <Hungarian>bekötöd a fogoly szemét</Hungarian>
+            <Russian>Вы завязываете пленнику глаза</Russian>
+            <Portuguese>Vendas os olhos</Portuguese>
```

Comment:
```suggestion
            <Portuguese>Vendar prisioneiro</Portuguese>
```

I know at least this translation is wrong. I don't like the idea of using Google Translate.

Reply:
I see your point, should I revert them?

Reply:
Keep the portugese one Grim suggested and the French one I did, remove the others.